### PR TITLE
v1.1.0 release

### PR DESCRIPTION
**Pull request by IAlibay**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,19 +18,20 @@ build:
...
     - coverage
     - networkx
     - numpy 
     - openff-models ~=0.1.2
     - openff-toolkit >=0.13
-    - openff-units >=0.2,<0.3
+    - openff-units !=0.2.3,>=0.2,<0.3
     - pint
     - pip
     - pydantic
```
Comment (mikemhenry) [inline recipe/meta.yaml:33]: Do we want to add the min version here to be `pydantic>=1.10.17`

https://docs.pydantic.dev/latest/migration/#using-pydantic-v1-features-in-a-v1v2-environment

File: recipe/meta.yaml [hunk 2]
```diff
@@ -61,4 +62,4 @@ extra:
     - dotsdl
```
Comment (mikemhenry) [inline recipe/meta.yaml:62]: We should also add @jthorton

**Comment by mikemhenry** [comment]
We should change this: https://github.com/conda-forge/gufe-feedstock/pull/39/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR56 to https://gufe.readthedocs.io/

File: recipe/meta.yaml (hunk 2 above)
Comment (mikemhenry) [reply recipe/meta.yaml:62]: Oh and @ianmkenney

Comment (IAlibay) [reply recipe/meta.yaml:62]: let's do this separately so we don't miss anyone - including Hannah

**Review by mikemhenry — APPROVED** [review]

**Pull request merged**